forge/ee/lib/sso/index.js

```diff
@@ -2,18 +2,92 @@ module.exports.init = async function (app) {
     // Set the SSO feature flag
     app.config.features.register('sso', true, true)
 
+    /**
+     * A temporary hardcoded list of SAML configurations.
+     * This will eventually either come from .yml or via a db table
+     */
+    const PROVIDERS = {
+        // provider1: {
+        //     id: 'provider1',
+        //     domainFilter: '^.*@flowforge.com$',
```

Comment:
I know this is commented out, but the `.` in `flowforge.com` should probably be escaped because as it is it represents any char. (Just something to think about when we validate customer supplied regex)

Reply:
Good point to keep in mind. This whole block will go away in the next PR when we figure out the right way to provide this configurations.